src/components/app/community/community-board.tsx

```
@@ -224,14 +229,15 @@ export default function CommunityBoard(props: CommunityBoardType & {
               margin-bottom: 2.3rem;
             `}
           >
-            <span
+            <div
+              id="content"
+              contentEditable={false}
+              dangerouslySetInnerHTML={{ __html: content }}
               css={css`
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)